src/Navbar.js

```diff
@@ -0,0 +1,9 @@
+import './Navbar.css';
+
+function Navbar() {
```

Comment:
Please change to es6 function for consistency.

Reply:
Changed!